src/python/foglamp/core/server_daemon.py

```diff
@@ -17,18 +17,18 @@
 import sys
 import time
 import daemon
+import setproctitle
 from daemon import pidfile
-
-from foglamp.core.server import Server
+from foglamp.core.server import Server, _STORAGE_SERVICE_NAME
 from foglamp import logger
 
 __author__ = "Amarendra K Sinha, Terris Linenbach"
 __copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
 __license__ = "Apache 2.0"
 __version__ = "${VERSION}"
 
-_PID_PATH = os.path.expanduser('~/var/run/foglamp.pid')
-_WORKING_DIR = os.path.expanduser('~/var/log')
+_PID_PATH =  os.getenv('FOGLAMP_PID_PATH', os.path.expanduser('~/var/run/foglamp.pid'))
+_WORKING_DIR = os.getenv('WORKING_DIR', os.path.expanduser('~/var/log'))
```

Comment:
We should not create this, it doesn't fit within the file structure as defined by Ivan